services/dumper.js

```diff
@@ -277,6 +283,69 @@ function Dumper(config) {
     });
   }
 
+  this.dumpFieldIntoModel = (tableName, field) => {
+    const newFieldText = handlebarsTemplate(`app/models/update/${config.dbDialect === 'mongodb' ? 'mongo' : 'sequelize'}-field.hbs`)({ field })
+      .trimRight();
+    const tableFileName = getTableFileName(tableName);
+
+    const currentContent = fs.readFileSync(tableFileName, 'utf-8');
+    const regexp = config.dbDialect === 'mongodb'
+      ? /(mongoose.Schema\({)/
+      : /(sequelize.define\(\s*'.*',\s*{)/;
+
+    if (regexp.test(currentContent)) {
+      const newContent = currentContent.replace(regexp, `$1\n${newFieldText}`);
+      writeFile(tableFileName, newContent, 'update');
+    } else {
+      logger.warn(chalk.bold(`WARNING: Cannot add the field definition ${field.name} \
+automatically. Please, add it manually to the file '${tableFileName}':\n${newFieldText}`));
+    }
+  };
+
+  this.dumpReferenceIntoModel = (tableName, reference) => {
+    const tableFileName = getTableFileName(tableName);
+    const referenceWithMetaDatas = getReferenceWithMetaDatas(reference, () => true);
+
+    const currentContent = fs.readFileSync(tableFileName, 'utf-8');
+    const regexpModelVariableName = /\s(\w+)\s*=\s*sequelize\s*\.\s*define/;
+    let modelVariableName;
+    if (regexpModelVariableName.test(currentContent)) {
+      const matches = currentContent.match(regexpModelVariableName);
+      [, modelVariableName] = matches;
+    } else {
+      modelVariableName = stringUtils.pascalCase(stringUtils.transformToSafeString(tableName));
+    }
+
+    const newFieldText = handlebarsTemplate('app/models/update/sequelize-relationship.hbs')({
+      reference: referenceWithMetaDatas,
+      modelVariableName,
+    })
+      .trimRight();
+
+    const regexpAssociate = /(\.associate\s*=\s*(function)?\s*\(models\)\s*(=>)?\s*{)/;
+
+    if (regexpAssociate.test(currentContent)) {
+      const newContent = currentContent.replace(regexpAssociate, `$1\n${newFieldText}`);
+      writeFile(tableFileName, newContent, 'update');
+    } else {
+      logger.warn(chalk.bold(`WARNING: Cannot add the field definition ${reference.name} \
+automatically. Please, add it manually to the file '${tableFileName}':\n${newFieldText}`));
```

Comment:
Maybe we could specify which file.

Reply:
hmmm not sure to follow; there is already `Please, add it manually to the file '${tableFileName}':\n${newFieldText}` 🤔